whatsupyoutube/main.py

```diff
@@ -29,36 +31,88 @@ def create_df(search_results) -> pd.DataFrame:
 def main() -> None:
     load_dotenv()
 
-    api_key = get_env_var("API_KEY")
 
-    channel_id = "UCVRobwJfa8nHMfJI3C8Rgdw"
-    page_token = ""
 
+    #api url components
+    api_key = get_env_var("API_Key")
+
+    channel_id = "UCX6OQ3DkcsbYNE6H8uQQuVA"
```

Comment:
We should pass this either at runtime or specify it as an env variable. For sake of simplicity right now, let's move this to be an environment variable.

Reply:
you're saying api_key -> env_var? wouldn't that cause readability issues as you go further

Reply:
I'm saying `channel_id` can be an environment variable.